nservicebus/gateway/scale-out.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Gateway And Scale-Out
+summary: Explanation of how the Gateway handles Scale-Out
+tags: []
+related:
+ - samples/gateway
+---
+
+The gateway is designed to run on a single instance for each logical endpoint. A scaled out endpoint needs one instance to receive all the incoming HTTP gateway traffic for that endpoint. There are different strategies to achieve this based on the transport and the scale-out mechanism utilized:
```

Comment:
> endpoint needs one instance

From this sentence it's not clear to me what is that one instance? Endpoint instance or gateway instance?

> all the incoming HTTP gateway traffic

Stupid question - is there any other incoming HTTP traffic for that endpoint? I'm not exactly sure what that means.